### PR TITLE
fix typo, improve html for module view

### PR DESCRIPTION
**Pull request by nworr**
improve #4804 

Funded by 3Liz

### CODE REVIEW

File: lizmap/modules/admin/templates/server_information.tpl [hunk 1]
```diff
@@ -37,17 +37,21 @@
...
             <th>{@admin.server.information.module@}</th>
             <th>{@admin.server.information.module.version@}</th>
         </tr>
+        </thead>
+        <tbody>
         {foreach $modules as $module}
             <tr>
-                <th style='width:20%;'> {$module->slug}</th>
-                <td style='width:20%;'>{$module->version}</td>
+                <th>{$module->slug}</th>
```
Comment (Gustry) [inline lizmap/modules/admin/templates/server_information.tpl:50]: Not sure why, I took it from L165, it was this.
Before, it was 30% from your initial commit I think.

File: lizmap/modules/admin/templates/server_information.tpl [hunk 2]
```diff
@@ -167,7 +171,7 @@
             {* Fixed in lizmap_server plugin 1.3.2 https://github.com/3liz/qgis-lizmap-server-plugin/commit/eb6a773ba035f877e9fa91db5ef87911a2648ee1 *}
             <th style="width:20%;">
                 {$version['name']}
-                {if $version['homepage']}
+                {if array_key_exists('homepage', $version)}
```
Comment (Gustry) [inline lizmap/modules/admin/templates/server_information.tpl:174]: Hum, now, the `homepage` is always provided, but the string can be empty I think.
```suggestion
                {if array_key_exists('homepage', $version) && $version['homepage']}
```
??

indeed, nice to check if the key exists anyway.

File: lizmap/modules/admin/templates/server_information.tpl (hunk 1 above)
Comment (nworr) [reply lizmap/modules/admin/templates/server_information.tpl:50]: I remove inline style, and prefer to use `table-server-info` which is used in all other tables of this page

File: lizmap/modules/admin/templates/server_information.tpl (hunk 2 above)
Comment (nworr) [reply lizmap/modules/admin/templates/server_information.tpl]: Home page is undefined for me : see jelix debug bar:
 image

Comment (Gustry) [reply lizmap/modules/admin/templates/server_information.tpl]: Ok, thanks.
So should we do both ? if the string is existing, but empty ?

**Review by Gustry — APPROVED** [review]
LGTM

Just E2E is failing, because of the new HTML table header

File: lizmap/modules/admin/templates/server_information.tpl (hunk 2 above)
Comment (nworr) [reply lizmap/modules/admin/templates/server_information.tpl]: yeah that's it, need to keep both.

**Pull request merged**